src/rest/DataUnionEndpoints.js

```diff
@@ -448,7 +450,9 @@ async function untilWithdrawIsComplete(client, getWithdrawTxFunc, getBalanceFunc
         log(`Waiting until sidechain AMB has collected required signatures for hash=${messageHash}...`)
```

Comment:
if the message is alreadySent, don't throw an error. Just print log line. This happens on xDai-mainnet bridge for ex.